Shelfie.Identity/Shelfie.Identity.BusinessLogic/UseCases/LoginUser/LoginUserCommandHandler.cs

```diff
@@ -17,12 +20,14 @@ public LoginUserCommandHandler(UserManager<IdentityUser> userManager, IJwtServic
         _jwtService = jwtService;
     }
 
-    public async Task<string?> Handle(LoginUserCommand request, CancellationToken cancellationToken)
+    public async Task<OperationResult<string?>> Handle(LoginUserCommand request, CancellationToken cancellationToken)
     {
+        var result = new OperationResult<string?>();
+
         var user = await _userManager.FindByNameAsync(request.Username);
         if (user == null || !await _userManager.CheckPasswordAsync(user, request.Password))
         {
-            return null;
+            return result;
```

Comment:
please consider creating two derivved classes SuccessfulResult and FailureResult or static methods OperationResult.Success() / OperationResult.Failure()